<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResampleDialog</class>
 <widget class="QDialog" name="ResampleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>143</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Resample options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Resample method</string>
       </property>
       <widget class="QRadioButton" name="radioButton_M_UD">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>131</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Upsample/downsample</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_M_NN">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>121</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Nearest neighbor</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Resample strategy</string>
       </property>
       <widget class="QRadioButton" name="radioButton_S_R">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>60</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Resample only</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_S_RO">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>191</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Resample, offset adjust</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_S_ROP">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>191</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Resample, offset adjust, pad right</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ResampleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ResampleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
